web/src/components/molecules/MyIntegration/MyIntegrationHeader/index.tsx

```diff
@@ -0,0 +1,17 @@
+import styled from "@emotion/styled";
+
+import PageHeader from "@reearth-cms/components/atoms/PageHeader";
+
+export type Props = {
+  title: string;
+  onBack?: () => void;
+};
+
+const MyIntegrationHeader: React.FC<Props> = ({ title, onBack }) => {
+  return <IntegrationPageHeader onBack={onBack} title={title} />;
+};
```

Comment:
Is the header's background a different color depending on page? If not, just put the background color in to the PageHeader atom and you can skip components like this.

Reply:
@KaWaite , it is.
On some pages (workspace settings for example), it is transparent and kept grey.

Reply:
Can you change it so that the PageHeader accepts either a backgroundColor prop or a styles prop?
Especially if we have multiple unique XxxxHeader components that just add a background color, adding the prop will allow us to clean up the files so we only have essentials.

Reply:
@KaWaite , done, I've added a background prop which is optional.